### PR TITLE
Fixing links to download app signing scripts

### PR DESCRIPTION
**Pull request by sadohert**
The links to scripts that are needed in these docs were not working correctly.  There was an extra `/` appended in the url. Updating this

### CODE REVIEW

**Comment by sadohert** [comment]
Ready for review

**Comment by sadohert** [comment]
@cwarnermm @justinegeffen  - Would one of you be the reviewers for this change?  Its more of a bug fix.  No real content change.

File: site/content/contribute/mobile/unsigned/android.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ With every Mattermost mobile app release, we publish the Android unsigned apk in
 3. [JQ](https://stedolan.github.io/jq/) is like sed for JSON data - you can use it to slice, filter, map, and transform structured data with the same ease that `sed`, `awk`, and `grep` let you work with text.
 4. Android SDK as described in the [Developer Setup](/contribute/mobile/developer-setup/#additional-setup-for-android).
 5. Setup keys and Google Services as described in steps 2, 3, 4 & 6 of the [Build your own App guide](/contribute/mobile/build-your-own/android/#build-preparations).
```
Comment (justinegeffen) [inline site/content/contribute/mobile/unsigned/android.md:16]:
```suggestion
5. Set up keys and Google Services as described in steps 2, 3, 4, and 6 of the [Build your own App guide](/contribute/mobile/build-your-own/android/#build-preparations).
```

File: site/content/contribute/mobile/unsigned/android.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ With every Mattermost mobile app release, we publish the Android unsigned apk in
...
-6. `sign-android` script to sign the Android app.
+6. [sign-android](/scripts/sign-android) script to sign the Android app.
```
Comment (justinegeffen) [inline site/content/contribute/mobile/unsigned/android.md:17]: Is this a link to a file or a section in the docs?

File: site/content/contribute/mobile/unsigned/android.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ Options:
 Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should really be using your own `package identifier`, `google services settings` and maybe change the `display name`.
 
 * Create a folder that will serve as your working directory to store all the needed files.
-* Download the [sign-android](/scripts/sign-android/) script and save it in your working directory.
+* Download the [sign-android](/scripts/sign-android) script and save it in your working directory.
 * Download the [Android unsigned build](https://github.com/mattermost/mattermost-mobile/releases) and save it in your working directory.
 * Open a terminal to your working directory and make sure the `sign-android` script it is executable.
```
Comment (justinegeffen) [inline site/content/contribute/mobile/unsigned/android.md:52]:
```suggestion
* Open a terminal to your working directory and make sure the `sign-android` script is executable.
```

File: site/content/contribute/mobile/unsigned/ios.md [hunk 4]
```diff
@@ -60,7 +60,7 @@ and you should be using your own `provisioning profiles`, `certificate`, also yo
 * Download your **Apple Distribution certificate** from the [Apple Developer portal](https://developer.apple.com/account/resources/certificates/list) and save it in your working directory.
 * Install the previously downloaded certificate into your macOS Keychain. [Learn more](https://developer.apple.com/support/certificates).
 * Download your **Provisioning profiles** from the [Apple Developer portal](https://developer.apple.com/account/resources/profiles/list) and save it in your working directory.
-* Download the [sign-ios](/scripts/sign-ios/) script and save it in your working directory.
+* Download the [sign-ios](/scripts/sign-ios) script and save it in your working directory.
 * Download the [iOS unsigned build](https://github.com/mattermost/mattermost-mobile/releases) and save it in your working directory.
 * Open a terminal to your working directory and make sure the `sign-ios` script it is executable.
```
Comment (justinegeffen) [inline site/content/contribute/mobile/unsigned/ios.md:65]:
```suggestion
* Open a terminal to your working directory and make sure the `sign-ios` script is executable.
```

File: site/content/contribute/mobile/unsigned/ios.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ With every Mattermost mobile app release, we publish the iOS unsigned ipa in in
 	```bash
 	$ xcode-select --install
 3. Set up your Certificate and Provisioning profiles as described in steps 1 & 2 for [Run on iOS Devices](/contribute/mobile/developer-setup/run/#run-on-ios-devices) in the Developer Setup.
```
Comment (justinegeffen) [inline site/content/contribute/mobile/unsigned/ios.md:16]:
```suggestion
3. Set up your Certificate and Provisioning profiles as described in steps 1 and 2 for [Run on iOS Devices](/contribute/mobile/developer-setup/run/#run-on-ios-devices) in the Developer Setup.
```

**Review by justinegeffen — COMMENTED** [review]
Thanks, Stu! Some other comments/edits added which I picked up when reviewing.

File: site/content/contribute/mobile/unsigned/android.md [hunk 6]
```diff
@@ -14,7 +14,7 @@ With every Mattermost mobile app release, we publish the Android unsigned apk in
 3. [JQ](https://stedolan.github.io/jq/) is like sed for JSON data - you can use it to slice, filter, map, and transform structured data with the same ease that `sed`, `awk`, and `grep` let you work with text.
```
Comment (cwarnermm) [inline site/content/contribute/mobile/unsigned/android.md:14]:
```suggestion
3. [JQ](https://stedolan.github.io/jq/) is like `sed` for JSON data - you can use it to slice, filter, map, and transform structured data with the same ease that `sed`, `awk`, and `grep` let you work with text.
```

File: site/content/contribute/mobile/unsigned/android.md [hunk 7]
```diff
@@ -47,7 +47,7 @@ Options:
 Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should really be using your own `package identifier`, `google services settings` and maybe change the `display name`.
```
Comment (cwarnermm) [inline site/content/contribute/mobile/unsigned/android.md:47]:
```suggestion
Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should use your own `package identifier`, `google services settings` and change the `display name`.
```

**Comment by cwarnermm** [comment]
There's a small typo in this line of the document: "With every Mattermost mobile app release, we publish the iOS unsigned ipa in **in** the [GitHub Releases](https://github.com/mattermost/mattermost-mobile/releases) page" (**bolded**) that I was unable to flag to fix. (Not sure why, but likely related to the content being pre-existing and unchanged.)

**Review by cwarnermm — APPROVED** [review]
Minor suggestions proposed inline.

File: site/content/contribute/mobile/unsigned/android.md [hunk 8]
```diff
@@ -44,12 +44,12 @@ Options:
 
 #### Sign the Mattermost Android app
 
-Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should really be using your own `package identifier`, `google services settings` and maybe change the `display name`.
+Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should use your own `package identifier`, `google services settings` and change the `display name`.
```
Comment (justinegeffen) [inline site/content/contribute/mobile/unsigned/android.md:47]:
```suggestion
Now that all requirements are met, it's time to sign the Mattermost app for Android. Most of the options of the signing tool are optional but you should use your own `package identifier`, `google services settings`, and change the `display name`.
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Looks good to me. I'll verify that the link to the script is working on live (it's not showing in the preview for some reason).

**Pull request merged**